### PR TITLE
Moved ots and grav from ert-statoil

### LINKED ISSUE #451: Move OTS and grav code from libecl to libres
Needs #450

### PR DESCRIPTION
**Pull request by stefoss23**
**Issue**
Resolves #451

**Pre un-WIP checklist**
- [ ] Have completed graphical integration test steps

**Depends on**
* Statoil/libecl#

### CODE REVIEW

**Comment by joakim-hove** [comment]
Put both the `grav` and `ots` packages as subpackages in `res/fm/geophysics`

**Comment by joakim-hove** [comment]
The job description file: https://github.com/Statoil/ert-statoil/blob/master/etc/ERT/Config/jobs/seismic/OVERBURDEN_TS should be moved to 'share/forward-models/geophysics` in libres.

**Comment by joakim-hove** [comment]
OK - with this libres depends on `segyio` - should:

1. Add `segyio` to the `requirements.txt` file.
2. Make sure `segyio` is installed (might require `pip install`) in the `travis/install_python_packages.py` script [here](https://github.com/Statoil/ert/blob/master/travis/install_python_packages.py).

Apart from the new segyio dependency this should be ready to be merged.

@markusdregi - opinions?

**Comment by joakim-hove** [comment]
Looks like you have forgotten the `OVERBURDEN_TS` file from the comment above? In addition an entry:
```
INSTALL_JOB_DIRECTORY forward-models/geophysics
```
should be added to the libres site-config file.

File: share/ert/forward-models/geophysics/OVERBURDEN_TS [hunk 1]
```diff
@@ -0,0 +1,10 @@
+EXECUTABLE       Scripts/overburden_timeshift
```
Comment (joakim-hove) [inline share/ert/forward-models/geophysics/OVERBURDEN_TS:1]: Does the script `Scripts/overburden_timeshift` exist?

**Comment by stefoss23** [comment]
In absence of more comments, this is finished. The TRAVIS failure is due to recent erronous ert-(stress)merges.

**Comment by joakim-hove** [comment]
Ok

**Comment by andreabrambilla** [comment]
Is anything happening in this PR or shall we just close it?

**Comment by stefoss23** [comment]
Close it. 
It was moved into another PR eons ago.

**Pull request closed without merging**